package.json

```diff
@@ -298,7 +298,7 @@
     "build-shared": "yarn && webpack --config webpack.shared.config.js",
     "build-static-viz": "yarn && yarn build:cljs && webpack --config webpack.static-viz.config.js",
     "build-static-viz:watch": "webpack --config webpack.static-viz.config.js --watch",
-    "precommit": "lint-staged",
+    "precommit": "lint-staged && yarn validate-e2e-filenames --staged",
```

Comment:
I wonder if we can actually use [`lint-staged`](https://github.com/okonet/lint-staged) here? We use it to run prettier, and eslint against _staged_ files. Maybe we can avoid retrieving a list of staged files on our own and delegate it to `lint-staged`?

Reply:
We most likely can, but the script needs to know about the staged files somehow so we'd need to pipe the output to it. Wouldn't that more convoluted?

Reply:
Yep, I've read the documentation and it states that the command will be run with all files as arguments, but I can't get it to work.